src/extension.ts

```diff
@@ -149,6 +154,89 @@ async function createMongoDatabase(server: IMongoServer) {
 	}
 }
 
+async function createDocDBDatabase(server: CosmosDBResourceNode) {
+	const databaseName = await vscode.window.showInputBox({ placeHolder: 'Database Name' });
+	if (databaseName) {
+		const masterKey = await server.getPrimaryMasterKey();
+		const endpoint = await server.getEndpoint();
+		let client = new DocumentClient(endpoint, { masterKey: masterKey });
+		client.createDatabase({ id: databaseName }, function (err, created) {
+			if (!err) {
+				vscode.window.showInformationMessage("Created a db with name " + databaseName);
+			} else {
+				vscode.window.showErrorMessage(err.body);
+				console.log(err.body);
+			}
+			explorer.refresh(server);
+		}
+		);
+	}
+}
+
+async function createDocDBCollection(db: DocDBDatabaseNode) {
+	const collectionName = await vscode.window.showInputBox({ placeHolder: 'Collection Name' });
+	if (collectionName) {
+		let masterKey = db.getPrimaryMasterKey();
+		let endpoint = db.getEndpoint();
+		let options = {};
+		let partitionKey: string = await vscode.window.showInputBox({
+			prompt: 'Partition Key: Choose a JSON property name that will possibly have a wide range of values.',
```

Comment:
I need to remove the trailing period for consistency.

Reply:
too long